content/product/iac.md

```diff
@@ -0,0 +1,241 @@
+        number: "10,000s"
+        description: of community members
+    company:
+        number: "1,000s"
+        description: of companies
+    integration:
+        number: "150+"
+        description: Cloud and service integrations
+
+key_features_below:
+    items:
+        - title: "The fastest and easiest way to use Pulumi IaC at scale"
+          sub_title: "Pulumi Cloud"
+          description: |
+             A fully-managed service for Pulumi IaC plus so much more. Manage and store infrastructure state & secrets, collaborate within teams, view and search infrastructure, and manage security and compliance using Pulumi Cloud.
+          image: "/images/product/pulumi-cloud-dashboard.png"
+          button:
+            text: "Learn more about Pulumi Cloud"
+            link: "/product/pulumi-service/"
+          features:
+              - title: Pulumi IaC
+                description: |
+                    Utilize open-source IaC in TypeScript, Python, Go, C#, Java and YAML. Build and distribute reusable components for 150+ cloud & SaaS providers.
+              - title: Automate deployment workflows
+                description: |
+                    Orchestrate secure deployment workflows through GitHub or an API.
+              - title: Search and analytics
+                description: |
+                    View resources from any cloud in one place. Search for resources across clouds with simple queries and filters.
+              - title: Pulumi Automation API
+                description: |
+                    Build custom deployment and CI/CD workflows that integrate with Pulumi Developer Portal, custom portals, or CLIs.
+              - title: Developer portals
+                description: |
+                    Create internal developer portals to distribute infrastructure templates using Pulumi or the Backstage-plugin.
+              - title: State & secrets
+                description: |
+                    Securely store state with built-in secrets manager or bring your own KMS.
+              - title: Identity and access control
+                description: |
+                    Manage teams with SCIM, SAML SSO, GitHub, GitLab, or Atlassian. Set permissions and access tokens.
+              - title: Policy enforcement
+                description: |
+                    Build policy packs from 150 policies or write your own. Leverage compliance-ready policies for any cloud to increase compliance posture and remediation policies to correct violations.
+              - title: Audit logs
+                description: |
+                    Track and store user actions and change history with option to export logs.
+
+get_started:
+    title: Getting started
+
+    get_started:
+        title: Get started now
+        description: |
+            Deploy your first app in just five minutes. Follow our tutorials for AWS, Azure, Google Cloud, Kubernetes, and more.
+        cta_text: Get Started
+
+    migrate:
```

Comment:
there is a slight formatting error

![image](https://github.com/user-attachments/assets/d132218e-1d1f-4f14-bbe8-264d47f202ff)

Reply:
@shughes26 Can you help with this?